host/xtest/pkcs11_1000.c

```diff
@@ -309,6 +309,36 @@ static void xtest_pkcs11_test_1001(ADBG_Case_t *c)
 		if (!ADBG_EXPECT_CK_OK(c, rv))
 			goto out;
 
+		/**
+		 * Verify that fields with CK_UNAVAILABLE_INFORMATION gets
+		 * correct value
```

Comment:
I think it would be better to highlight here that the pkcs11 always return CK_UNAVAILABLE_INFORMATION for `ulMaxSessionCount`, `ulMaxRwSessionCount`, `ulTotalPublicMemory` and `ulFreePublicMemory` so we can test them safely.

As per `ulTotalPrivateMemory`  and `ulFreePrivateMemory` I think we can implement them one day, so I would not enforce there values in this test.

Reply:
Ok